### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by roger10cassares**

### CODE REVIEW

File: FunctionalComponentExample.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+} from 'react-native';
+
+import QRCodeScanner from 'react-native-qrcode-scanner';
+
+
+const ScanQRCodeScreen = () => {
+  const [qrcodeGetData, setQRCodeGetData] = useState(null);
+  
+  useEffect(() => {
+  },[qrcodeGetData]);
```
Comment (javatheist) [inline FunctionalComponentExample.js:20]: What is the purpose of an empty effect?

Comment (roger10cassares) [reply FunctionalComponentExample.js:20]: Hi! From my understanding is to provide a realtime update on the screen when the value betewwn `[]` changes.

If the `useEffect ` Hook is not applied here, the change in `qrcodeGetData ` will be updated just after the next re-render.

Comment (renatobenks) [reply FunctionalComponentExample.js:20]: it's not actually true as long as React does that for you already. Once you have your state updated, the component will immediately re-render by himself

**Review by renatobenks — COMMENTED** [review]
is this example really need? I think the primary example should be using component functional-like by default

**Comment by roger10cassares** [comment]
Ho @renatobenks 

Without this effect I was not able to update the useState Hook variable. Just after another reading from qrcode scanner.

The example at README.md was a class component based. I guess the functional example could makes it lighter, faster and simple. Since I have never seen the this package with functional-like I have proposed this one. 🙂

If you have a better code feel free to complement! 

Many thanks!

File: FunctionalComponentExample.js (hunk 1 above)
Comment (roger10cassares) [reply FunctionalComponentExample.js:20]: HI! It just not worked for me. COuld you post an exemple? I mean it not work at the same time the `setQRCodeGetData` was executed

Comment (renatobenks) [reply FunctionalComponentExample.js:20]: what exactly didn't work? once you're reading a QR code, the `onSuccess` function has been called without `useEffect`?

**Comment by renatobenks** [comment]
@roger10cassares could you move it to `examples/` folder instead of putting it on root?

I reconsidered what I just said about the purpose of this example because we definitely need to rethink how all examples work and even how are examples, but we don't need to do this now

**Pull request closed without merging**